lib/interfaces/bdd.js

```diff
@@ -91,6 +91,20 @@ module.exports = function(suite){
     };
 
     /**
+     * Do not randomize.
+     */
+    context.describe.noRandom = function func(title, fn) {
```

Comment:
this behavior seems a bit overly complicated.  I think I'd prefer if we dropped it for now and add it if people are complaining